applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -17,93 +17,22 @@ if TRANSPORT_BIS
 rsource "bt_stream/broadcast/Kconfig"
 endif # TRANSPORT_BIS
 
+if TRANSPORT_CIS
+rsource "bt_stream/unicast/Kconfig"
+endif # TRANSPORT_CIS
+
 config CODEC_CAP_COUNT_MAX
```

Comment:
Maybe we should add a QoS menu to the unicast Kconfig

Reply:
Some of them are used, some are not, so that's why I didn't move them. There might be a need for capabilities when we use scan_delegator, but I'm not sure. Most of the QoS stuff are used in le_audio.h though